### PR TITLE
made event bus url a url parameter as well

### PR DESCRIPTION
**Pull request by edewit**
@vorburger no longer hardcoded to locahost:8080

### CODE REVIEW

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java [hunk 1]
```diff
@@ -39,23 +39,28 @@
...
 
     private final TokenProvider tokenProvider;
 
     public LoginCommand(TokenProvider tokenProvider) {
         this.tokenProvider = tokenProvider;
-        String jsURL = System.getProperty("storeys.jsURL");
-        if (jsURL != null) {
-            scratchJSExtensionURL = jsURL;
+        scratchJSExtensionURL = eventBusUrl = getSystemPropertyOrDefault("storeys.jsURL", scratchJSExtensionURL);
+        eventBusUrl = getSystemPropertyOrDefault("storeys.eventBusUrl", eventBusUrl);
```
Comment (vorburger) [inline web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java]: storeys.eventBusURL (upper case) here as well

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java [hunk 2]
```diff
@@ -39,23 +39,28 @@
...
- *
- * @author edewit
  */
 public class LoginCommand implements Command {
 
-    private static final String URL_PREFIX = "http://scratchx.org/?url=%s&code=%s";
+    private static final String URL_PREFIX = "http://scratchx.org/?url=%s&code=%s&evenbusUrl=%s";
 
     private String scratchJSExtensionURL = "https://rawgit.com/vorburger/minecraft-storeys-maker/master/scratch/minecraft.js";
+    private String eventBusUrl = "http://localhost:8080/eventbus";
```
Comment (vorburger) [inline web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java]: eventBusURL (for consistency with scratchJSExtensionURL)

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java [hunk 3]
```diff
@@ -39,23 +39,28 @@
...
 
     private final TokenProvider tokenProvider;
 
     public LoginCommand(TokenProvider tokenProvider) {
         this.tokenProvider = tokenProvider;
-        String jsURL = System.getProperty("storeys.jsURL");
-        if (jsURL != null) {
-            scratchJSExtensionURL = jsURL;
+        scratchJSExtensionURL = eventBusUrl = getSystemPropertyOrDefault("storeys.jsURL", scratchJSExtensionURL);
```
Comment (vorburger) [inline web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java]: am I missing something, or did that " = eventBusUrl = " slip in here unintentionally?

**Comment by vorburger** [comment]
@edewit actually I need this like right NOW :smile_cat: for #19 so I'll merge and fix up myself; hope OK.

**Comment by edewit** [comment]
sure didn't notice you use URL everywhere

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java (hunk 3 above)
Comment (edewit) [reply web/src/main/java/ch/vorburger/minecraft/storeys/web/LoginCommand.java]: ooops, it was late ;)

**Pull request merged**